training-react/tic-tac-toe/src/constants/propTypes.js

```diff
@@ -0,0 +1,8 @@
+import PropTypes from 'prop-types';
+
+export const infoUserPropType = PropTypes.shape({
```

Comment:
Move this file to `src/propTypes` following this. https://github.com/Wolox/tech-guides/blob/master/react/style-guide.md#folder-structure